plugins/auth-core.yml

```diff
@@ -1,7 +1,7 @@
 id: auth-core
 name: NestJS Auth Module
 description: Add the core functionality for authentication and authorization to your service. This plugin must be installed with one of the Auth Provider plugins.
-repo: amplication/plugins
+repo: https://github.com/amplication/plugins/tree/master/plugins/auth-core
```

Comment:
also here, do we need to set a branches tree folder ?

Reply:
it is there for all the others plugin, we might need to default it to the root level, right?